app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
On the root folder of Shopistant, you already have .gitignore file then why did you add this again in app folder???